packages/pg-vector-selector/src/AgenticaPgVectorSelector.ts

```diff
@@ -54,7 +54,7 @@ function getRetry(count: number) {
       }
     }
 
-    if (lastError) {
+    if (lastError != null) {
```

Comment:
lastError is null or Error, so add comparing with null

FYI: You can compare with null/undefined by `==null` syntax
https://basarat.gitbook.io/typescript/recap/null-undefined